go/vt/sqlparser/ast.go

```diff
@@ -725,6 +725,15 @@ func (node *TableName) Equal(t *TableName) bool {
 	return node.Name == t.Name && node.Qualifier == t.Qualifier
 }
 
+// ToViewName returns a TableName acceptable for use as a VIEW. VIEW names are
+// always lowercase, so ToViewName lowercasese the name and qualifier.
+func (node *TableName) ToViewName() *TableName {
+	return &TableName{
+		Name:      NewTableIdent(strings.ToLower(node.Name.v)),
+		Qualifier: NewTableIdent(strings.ToLower(node.Qualifier.v)),
```

Comment:
One last nit. I think this should not be lowered because qualifiers are case-sensitive. That's what I was trying to say in my earlier comment.